### ISSUE TITLE
dict.setdefault() does not work on **kwargs

### ISSUE THREAD

**Issue by rayluo** [brython-dev/brython#2220]
Bonjour, @PierreQuentel. :-)

This is my second bug report on setdefault. The [first one](https://github.com/brython-dev/brython/issues/1598) was quickly fixed 2 years ago. Perhaps I just love to use `setdefault()` more than others. :-D  Recently I ran into similar issue when using `**kwargs`.

```python
d = {'foo': 1}
# The following 4 all work as expected
print(d)
print(d.setdefault("foo", 2))
print(d.setdefault("bar", 2))
print(d)

def greet(**kwargs):
    kwargs.setdefault("name", "Unknown")  # Errors out
    print("Hello", kwargs)

greet()
```

Again, this is not an urgent bug. I'll leave it to you to decide when or whether to fix it. :-)

**Comment by PierreQuentel** [comment]
Good to hear from you Ray, thanks for the report !

**Comment by rayluo** [comment]
Thanks for the quick response, @PierreQuentel . So, this fix will be released in a future Brython 3.11.4? That is great. I can wait.

At the meantime, when trying to help test your fix

```html
<body id="body" onload="brython()" class="container"></body>

<script src="https://raw.githack.com/brython-dev/brython/master/www/src/brython.js" crossorigin="anonymous"></script>
<script src="https://raw.githack.com/brython-dev/brython/master/www/src/brython_stdlib.js" crossorigin="anonymous"></script>

<script type="text/python" src="bar.py"></script>  <!-- ERROR on this line -->
```

I ran into this strange error:
> TypeError: $B.scripts[filename] is undefined

It seems like the develop version of Brython somehow does not load a standalone python source file.

It has been a long time since I switch to the develop version. Did I forget any setup steps, or is this an unrelated new bug?
